uco-master/uco.ttl

```diff
@@ -41,3 +41,27 @@
 		;
 	.
 
+<http://unifiedcyberontology.org/observable#CyberAction>
+	rdfs:subClassOf <http://unifiedcyberontology.org/action#Action> ;
+	.
+
+<http://unifiedcyberontology.org/observable#CyberItem>
+	rdfs:subClassOf <http://unifiedcyberontology.org/core#Item> ;
+	.
+
+<http://unifiedcyberontology.org/observable#WindowsPEFileHeader>
```

Comment:
None of the rest of these four classes should be subclass of Facet.
They are complexdatatypes that are only present within a WindowsPEBinaryFile facet.

Reply:
This makes me question the modeling of composite abstractions in UCO. Because these are component pieces of something that is an Observable they are not Observable? If a car is an Observable, the tires, bumpers, steering wheel, and engine, are not Observables?  

I'll trust you, but it doesn't make sense to me.

Reply:
I would be happy to discuss more verbally.

Reply:
I am going to change these back. This will be in the turtle-cleanup-two branch.